opensrp-anc/build.gradle

```diff
@@ -178,7 +178,7 @@ tasks.withType(Test) {
 
 dependencies {
     implementation 'androidx.appcompat:appcompat:1.3.1'
-    implementation('org.smartregister:opensrp-client-native-form:2.1.17-SNAPSHOT@aar') {
+    implementation('org.smartregister:opensrp-client-native-form:2.1.19-PREVIEW-SNAPSHOT@aar') {
```

Comment:
Why are we using a preview version for NativeForms?

Reply:
the latest translation changes are currently in the preview version as the translation branch is not merged yet that is why preview version is used